Hyperbolic_surface_triangulation_2/include/CGAL/Complex_number.h

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2024
+// INRIA Nancy (France), and Université Gustave Eiffel Marne-la-Vallée (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
```

Comment:
As this will sooner or later go into the Number_types package it should have the LGPL header.